src/shared/lib/country-levels.js

```diff
@@ -0,0 +1,89 @@
+import path from 'path';
+import { folderFromZipURL } from './fetch/utils.js';
+import { readJSON } from './fs.js';
+import * as geography from './geography/index.js';
+
+const LEVELS = ['iso1', 'iso2', 'fips'];
+const TARGET_VERSION = 'v1.0.1';
+
+const levelCache = {};
+const countryLevelsDir = path.join('src', 'shared', 'vendor', 'country-levels');
+
+export function isId(str) {
+  if (!str) return false;
+  const [level, code] = str.split(':');
+  return LEVELS.includes(level) && !!code;
```

Comment:
NaBD but Boolean(code) is perhaps clearer

Reply:
Done